### ISSUE TITLE
Scrolling does not work reliably with complex child component

### ISSUE THREAD

**Issue by Snaptags** [dizco/react-scrollable-feed#7]
I am trying to use the scollable feed component using a single <VerticalTimeline> component as a child:
https://github.com/stephane-monnot/react-vertical-timeline

It is working perfectly as long as I set the forceScroll property, but removing it the feed only scrolls one or two times before stopping it. After manually scrolling to the bottom it resumes auto-scrolling but stops again shortly afterwards. It looks like isViewable starts returning false for some reason. 

**To Reproduce**
I added the component to your default example:
https://github.com/Snaptags/react-scrollable-feed/tree/VerticalTimeline/example
Just do `npm install;npm start` to reproduce the issue.

**Expected behavior**
The content should auto-scroll while at the end of the list.

**Desktop:**
 - Windows 10
 - Chrome 71.0.3578.98

Any ideas?

**Comment by dizco** [comment]
Hey @Snaptags 

Thanks for your feedback, it's really appreciated!

I dug and I think I found the bug. It seems to be related to the way bounding rectangles are calculated... They contain float values and that sometimes made the `isViewable` function return false.

I'm thinking of adding an epsilon to that check, such that if we get values for both bounding rects like 381.25 and 381, it would be considered at the bottom. This could be customized through props to allow even more flexibility.

What do you think?

PS: In your example, make sure to change the `changeDetectionFilter`, because right now it will never return true. Here's something you could do :
```tsx
changeDetectionFilter={(previousProps, newProps) => {
  const prevChildren = previousProps.children.props.children;
  const newChildren = newProps.children.props.children;

  return prevChildren !== newChildren
    && prevChildren[prevChildren.length - 1] !== newChildren[newChildren.length - 1];
}}
```

**Comment by dizco** [comment]
@Snaptags have a shot at v1.0.3 and let me know!

**Comment by Snaptags** [comment]
@dizco thanks for this fix, it indeed solves my problem with using _react-vertical-timeline_! Good work adding the parameter to fine-tune the setting on demand.
I should have removed the `changeDetectionFilter` from the example I provided, because I don't use it at all in my code. It often changes the content of existing items, i.e. the number of items is not what matters when looking for changes. I implemented my component to re-render when the input changes and _react-scrollable-feed_ seems to work fine using the default filter always returning true :-)
